source/MaterialXView/CMakeLists.txt

```diff
@@ -97,3 +100,11 @@ install(TARGETS MaterialXView
     RUNTIME DESTINATION bin)
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_BUILD_TYPE}/MaterialXView.pdb"
     DESTINATION "${CMAKE_INSTALL_PREFIX}/bin/" OPTIONAL)
+    
+# Copy over libraries and resources to local build area
```

Comment:
It would be better if we can avoid this duplication, and directly access the canonical data libraries and resources from the viewer.  One benefit of this approach is in shader development, where it's possible to modify the canonical data libraries in place and hot-reload them into the viewer, rather than maintaining two separate copies that can drift out of sync.

Reply:
I actually have another cmake change to use a common location for all binaries. Can omit this copy for now.